subliminal/providers/hosszupuska.py

```diff
@@ -0,0 +1,233 @@
+    def __str__(self):
+        subtit = "Subtitle id: " + str(self.subtitle_id) \
+               + " Series: " + self.series \
+               + " Season: " + str(self.season) \
+               + " Episode: " + str(self.episode) \
+               + " Release_group: " + str(self.release_group) + " "
+        if self.format:
+            subtit = subtit + " Format: " + self.format + " "
+        if self.resolution:
+            subtit = subtit + " Resolution: " + self.resolution
+        if self.date:
+            subtit = subtit + " Year: " + self.year
+        if six.PY3:
+            return subtit
+        return subtit.encode('utf-8')
+
+    def __init__(self, language, page_link, subtitle_id, series, season, episode,
+                 format, release_group, resolution, date):
+        super(HosszupuskaSubtitle, self).__init__(language, page_link=page_link)
+        self.subtitle_id = subtitle_id
+        self.series = series
+        self.season = season
+        self.episode = episode
+        self.format = format
+        self.release_group = release_group
+        self.resolution = resolution
+        self.date = date
+        self.year = None
+        if self.date:
+            self.year = date.strip().split('-')[0]
+
+    @property
+    def id(self):
+        return str(self.subtitle_id)
+
+    def get_matches(self, video):
+        matches = set()
+        # series
+        if video.series and sanitize(self.series) == sanitize(video.series):
+            matches.add('series')
+        # season
+        if video.season and self.season == video.season:
+            matches.add('season')
+        # episode
+        if video.episode and self.episode == video.episode:
+            matches.add('episode')
+        # year
+        if self.year and video.year and str(video.year) == str(self.year):
+            matches.add('year')
+        # resolution
+        if video.resolution and self.resolution and video.resolution == self.resolution:
+            matches.add('resolution')
+        # release_group
+        if (video.release_group and self.release_group and
+                any(r in sanitize_release_group(self.release_group)
+                    for r in get_equivalent_release_groups(sanitize_release_group(video.release_group)))):
+                        matches.add('release_group')
+        # other properties
+        if video.format and self.format and video.format.lower() == self.format.lower():
+            matches.add('format')
+
+        return matches
+
+
+class HosszupuskaProvider(Provider):
+    """Hosszupuska Provider."""
+    languages = {Language('hun', 'HU')} | {Language(l) for l in [
+        'hun', 'eng'
+    ]}
+    video_types = (Episode,)
+    server_url = 'http://hosszupuskasub.com/'
+
+    def initialize(self):
+        self.session = Session()
+        # self.session.headers['User-Agent'] = 'Subliminal/%s' % __short_version__
+
+    def terminate(self):
+        self.session.close()
+
+    def GetLanguage(self, text):
```

Comment:
I see no reason to mix conventions and use CamelCase in two methods while all the others follow the expected snake_case convention

Reply:
Thanks, for the comment. 
I corrected them.